genesyscloud/webdeployments_configuration/utils/resource_genesyscloud_webdeployments_configuration_messenger.go

```diff
@@ -69,15 +69,13 @@ func buildAppKnowledge(knowledge []interface{}) *platformclientv2.Knowledge {
 			Id: &knowledgeBaseId,
 		}
 	}
-
 	return ret
 }
 
 func buildMessengerApps(apps []interface{}) *platformclientv2.Messengerapps {
 	if len(apps) < 1 {
 		return nil
 	}
-
 	app := apps[0].(map[string]interface{})
 	return &platformclientv2.Messengerapps{
 		Conversations: buildAppConversations(app["conversations"].([]interface{})),
```

Comment:
If `app["conversations"]` is nil because the user didn't define it at all, then this line will throw the error `panic: interface conversion: interface {} is nil, not []interface {}`

So I would define your `&platformclientv2.Messengerapps` object as a variable `m`, set these fields conditionally 

```golang
if c, ok := app["conversations"].([]interface{}); ok {
  m.Conversations = c
}

// same thing for knowledge

return m
```